distribution/config.go

```diff
@@ -140,7 +140,7 @@ func (s *imageConfigStore) Get(d digest.Digest) ([]byte, error) {
 	return img.RawJSON(), nil
 }
 
-func (s *imageConfigStore) RootFSAndOSFromConfig(c []byte) (*image.RootFS, layer.OS, error) {
+func (s *imageConfigStore) RootFSAndOSFromConfig(c []byte) (*image.RootFS, string, error) {
```

Comment:
Why is this a method? Not even sure it has to be a public function as the functionality seems to be achievable with other API.

Reply:
This has always been a method, I purely extended it from returning just rootfs, to including rootfs and operating system. For example the `tempConfigStore` type implements this method, external to the distribution package.

Reply:
Ok, lets handle this in another PR. I thought this first appeared in an lcow pr.